doc/minversions.rst

```diff
@@ -1,13 +1,11 @@
 Minimum Supported GMT/Python/NumPy Versions
 -------------------------------------------
```

Comment:
Would this be too long?

```suggestion
Minimum Supported GMT/Python/NumPy/Pandas/Xarray Versions
---------------------------------------------------------
```

Reply:
Perhaps "Minimum Supported Versions" is enough.

Reply:
Changed to "Minimum Supported Versions" in f4049a09c9206c5b80cb47d56521453f1a16cbb4.